src/compiler/checker.ts

```diff
@@ -2424,12 +2424,15 @@ namespace ts {
             const visitedSymbolTables: SymbolTable[] = [];
             return forEachSymbolTableInScope(enclosingDeclaration, getAccessibleSymbolChainFromSymbolTable);
 
-            function getAccessibleSymbolChainFromSymbolTable(symbols: SymbolTable): Symbol[] | undefined {
+            /**
+             * @param {ignoreQualification} boolean Set when a symbol is being looked for through the exports of another symbol (meaning we have a route to qualify it already)

```

Comment:
It must remain optional because it's usage other than the direct call where it is set is in the callback position for `forEachSymbolTableInScope`. We want to pass `false` there anyway, so we keep is optional to avoid changing `forEachSymbolTableInScope` or creating another closure.